build.gradle

```diff
@@ -16,7 +16,6 @@ apply plugin: 'idea'
 apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: 'com.google.protobuf'
-apply plugin: 'checkstyle'
```

Comment:
I can't get checkstyle to work with my java version so I turned it off, but now it'd committing the build.gradle file as well.